libs/sr/page-team/src/TalentContent.tsx

```
@@ -71,27 +85,14 @@ export default function CharacterTalentPane() {
-            onClickTitle={() =>
-              // buildTc?.character
-              //   ? setBuildTc((buildTc) => {
-              //       if (buildTc.character) buildTc.character.constellation = i
-              //     })
-              //   :
-              database.chars.set(characterKey, {
-                eidolon: i === eidolon ? i - 1 : i,
-              })
-            }
-          />
+          <SkillDisplayCard talentKey={talentKey} sheetElement={sheetElement} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)